### ISSUE TITLE
Swiper Issue: only 1 widget with Slides/Carousel can be used per page - otherwise error

### ISSUE THREAD

**Issue by fleks** [elementor/elementor#14663]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

There are errors when using more then one widget of
- Image Carousel
- Media Carousel
- Reviews
- Slides
- Testimonial Carousel

on the same page or post.

The slides and elements can't be moved/swiped, nether with mouse nor with touch.

This is the error message:
![Swiper Error Message](https://flextensions.com/wp-content/uploads/2021/04/elmentor_swiper-issue_error-message.png)
```
Uncaught (in promise) TypeError: Cannot read property 'prototype' of undefined
at SwiperBC.createSwiperInstance (frontend.min.js?ver=3.2.1:2)
at frontend.min.js?ver=3.2.1:2
```

And here is the error causing part of frontend.min.js:
![Swiper Fail](https://flextensions.com/wp-content/uploads/2021/04/elmentor_swiper-issue_code-error.png)

Here are a few examples with different combinations (see related post at the end):
http://beta.flextensions.com/276/issue-v3-2-1-slider/

Tested in Chromium 89.0, Chrome 91.0 & Firefox 87.0

## Steps to reproduce

Insert at least two of the mentioned widget above (can be the same) in one page. 

EDIT: And activate Improved Asset Loading in Elementor > Settings > Experiments
As the widgets are from Elementor and the swiper.min.js is not affected by Improved Asset Loading I leave it as bug.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.46 (Unix)
	MySQL version: MySQL Community Server (GPL) v5.7.32
	PHP Version: 7.4.16
	PHP Max Input Vars: 4000
	PHP Max Post Size: 76M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.1
	Site URL: http://beta.flextensions.com
	Home URL: http://beta.flextensions.com
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 256M
	Permalink Structure: /%post_id%/%postname%/
	Language: en-US
	Timezone: 2
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.4.0-beta1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4469.4 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.2.1
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.2.1
		Author: Elementor.com

	Flexible Elementor Panel
		Version: 2.3.5.1
		Author: WebMat


== Elementor - Compatibility Tag ==
	
	Elementor Pro: incompatible
	Flexible Elementor Panel: header_not_exists

== Elementor Pro - Compatibility Tag ==
	


== Elements Usage ==
	
	section : 3
		accordion : 1
		alert : 1
		animated-headline : 1
		audio : 1
		blockquote : 2
		button : 1
		call-to-action : 1
		code-highlight : 4
		countdown : 1
		counter : 1
		divider : 1
		flip-box : 1
		form : 1
		gallery : 1
		google_maps : 1
		heading : 62
		icon : 1
		icon-box : 1
		icon-list : 1
		image : 1
		image-box : 1
		image-carousel : 1
		image-gallery : 1
		login : 1
		lottie : 1
		media-carousel : 1
		nav-menu : 1
		portfolio : 1
		posts : 1
		price-list : 1
		price-table : 1
		progress : 1
		reviews : 1
		share-buttons : 1
		sidebar : 1
		slides : 1
		social-icons : 1
		spacer : 1
		star-rating : 1
		table-of-contents : 1
		tabs : 1
		testimonial : 1
		testimonial-carousel : 1
		text-editor : 3
		toggle : 1
		video : 1
	wp-post : 12
		accordion : 1
		alert : 1
		animated-headline : 1
		audio : 1
		blockquote : 2
		button : 2
		call-to-action : 1
		code-highlight : 4
		countdown : 1
		counter : 1
		divider : 1
		ekit-nav-menu : 1
		elementskit-image-accordion : 1
		flip-box : 1
		form : 1
		gallery : 1
		google_maps : 1
		heading : 74
		icon : 1
		icon-box : 1
		icon-list : 1
		image : 9
		image-box : 1
		image-carousel : 5
		image-gallery : 1
		login : 1
		lottie : 1
		media-carousel : 3
		nav-menu : 1
		portfolio : 1
		posts : 1
		price-list : 1
		price-table : 1
		progress : 1
		reviews : 2
		share-buttons : 1
		sidebar : 1
		slides : 5
		social-icons : 1
		spacer : 1
		star-rating : 1
		table-of-contents : 1
		tabs : 1
		testimonial : 1
		testimonial-carousel : 2
		text-editor : 3
		text-path : 7
		toggle : 1
		video : 1
	wp-page : 2
	single-post : 1
		posts : 1
		theme-post-content : 1
		theme-post-title : 1


== Log ==
	
PHP: showing 4 of 4PHP: 2021-03-26 10:57:46 [notice X 2][/mnt/web521/b2/82/511189982/htdocs/flextensions.com/beta/wp-content/plugins/elementor/includes/api.php::160] Undefined index: pro_widgets [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-15 12:09:02 [notice X 1][/mnt/web521/b2/82/511189982/htdocs/flextensions.com/beta/wp-content/plugins/elementor/includes/api.php::160] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-22 19:09:14 [warning X 1][/mnt/web521/b2/82/511189982/htdocs/flextensions.com/beta/wp-content/plugins/elementor/core/files/manager.php::115] unlink(/mnt/web521/b2/82/511189982/htdocs/flextensions.com/beta/wp-content/uploads/elementor/css/post-8.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-22 19:09:14 [warning X 1][/mnt/web521/b2/82/511189982/htdocs/flextensions.com/beta/wp-content/plugins/elementor/core/files/manager.php::115] unlink(/mnt/web521/b2/82/511189982/htdocs/flextensions.com/beta/wp-content/uploads/elementor/css/post-21.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 202021-04-12 12:12:20 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-beta1',
  'to' => '3.2.0-beta2',
)]
2021-04-12 12:12:21 [info] elementor::elementor_updater Started 
2021-04-12 12:12:21 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Start  
2021-04-12 12:12:21 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Finished 
2021-04-12 12:12:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-beta1',
  'to' => '3.2.0-beta2',
)]
2021-04-12 12:12:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-beta1',
  'to' => '3.2.0-beta2',
)]
2021-04-15 13:14:09 [info] elementor::elementor_updater Started 
2021-04-15 13:14:09 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Start  
2021-04-15 13:14:09 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Finished 
2021-04-15 13:14:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-beta2',
  'to' => '3.2.0-beta3',
)]
2021-04-18 20:58:02 [info] elementor::elementor_updater Started 
2021-04-18 20:58:02 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Start  
2021-04-18 20:58:02 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-beta3',
  'to' => '3.2.0-beta4',
)]
2021-04-18 20:58:02 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Finished 
2021-04-18 20:58:02 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-beta3',
  'to' => '3.2.0-beta4',
)]
2021-04-22 19:08:57 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-beta4',
  'to' => '3.2.1',
)]
2021-04-22 19:09:11 [info] elementor::elementor_updater Started 
2021-04-22 19:09:12 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Start  
2021-04-22 19:09:13 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Finished 
2021-04-22 19:09:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-beta4',
  'to' => '3.2.1',
)]
```
</details>

**Comment by DorShahar** [comment]
Hi @fleks 
Thanks for reporting. I was able to replicate the bug and we will investigate this issue.

**Comment by adrianchinghc** [comment]
Facing the same issue here on my site. Had to downgrade to 3.1.4 for now.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.2.2

Feel free to update

Thanks!
